src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -2644,6 +2729,7 @@ export class ChatWidget extends Disposable implements IChatWidget {
 			welcomeOffset = Math.max(welcomeOffset - this.input.editSessionWidgetHeight, 0);
 		}
 		welcomeOffset = Math.max(welcomeOffset - this.input.attachmentsHeight, 0);
+		welcomeOffset = Math.max(welcomeOffset - chatSuggestNextWidgetHeight, 0);
```

Comment:
I don't think this is needed since suggest widget is never rendered when welcome is shown